pkg/logger/logger_test.go

```diff
@@ -8,38 +8,12 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/fatih/color"
+	log "github.com/charmbracelet/log"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/cloudposse/atmos/pkg/schema"
 )
 
-func captureOutput(f func()) string {
```

Comment:
Why are we dropping?

Reply:
@aknysh do you know why we might have done it this way as opposed to the simpler way that @Cerebrovinny proposes?